### PR TITLE
feat: add DashboardContainerWidgetInfo that has widgetKey

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

- Use DashboardContainerWidgetInfo in DashboardWidgetContainer.vue to use widgetKey as widget element's id not to initiate every time when drag & drop occurs.

https://user-images.githubusercontent.com/26986739/210030891-4a3bfe0b-72d6-4d75-ae96-91c3dbfcefa4.mov

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue [hunk 1]
```diff
@@ -5,8 +5,8 @@
         <template v-for="(width, idx) in widgetWidthList">
             <component :is="getWidgetComponent(widgetInfoList[idx].widget_name)"
                        v-if="widgetInfoList[idx]"
-                       :id="widgetInfoList[idx].widget_name"
-                       :key="`widget-${widgetInfoList[idx].widget_name}-${idx}`"
+                       :id="widgetInfoList[idx].widgetKey"
+                       :key="widgetInfoList[idx].widgetKey"
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue:9]: praise: 👍

File: src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue [hunk 2]
```diff
@@ -174,14 +183,13 @@ export default defineComponent<Props>({
...
-            emit('update:loading', true);
             await Promise.allSettled(promises);
-            emit('update:loading', false);
         };
         expose({
             refreshAllWidget,
         });
-        onMounted(() => {
+        onMounted(async () => {
+            await store.dispatch('reference/loadAll');
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue:192]: question: DashboardDetailPage has `reference/loadAll` already. Should I delete it?

**Review by yuda110 — APPROVED** [review]
LGTM! I left a question!

File: src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue (hunk 2 above)
Comment (WANZARGEN) [reply src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue:192]: Oh, I already removed it!

Comment (yuda110) [reply src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue:192]: Oh I missed it😅 Thanks!

**Pull request merged**